package.json

```diff
@@ -27,9 +27,9 @@
     "normalize-scss": "5.0.3"
   },
   "devDependencies": {
-    "@blackbaud/skyux": "2.37.0",
-    "@blackbaud/skyux-builder": "1.30.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.5"
+    "@blackbaud/skyux": "2.48.0",
+    "@blackbaud/skyux-builder": "1.33.1",
```

Comment:
We'll need to bump the version of Builder to get this working in future builds; might as well update SKY UX, too.
```
"@blackbaud/skyux": "2.48.3",
"@blackbaud/skyux-builder": "1.34.1",
"@skyux-sdk/builder-plugin-skyux": "1.0.0"
```

Reply:
Or, we could merge the following PR, first. Up to you.
https://github.com/blackbaud/skyux-theme/pull/61

Reply:
Merged the other PR, so this LGTM.